app/models/invenio_rdm_record_converter.rb

```diff
@@ -245,20 +250,21 @@ def subjects_for_scheme(terms, scheme)
     end
   end
 
-  def additional_titles(titles)
```

Comment:
haha, seems like I was the one who wrote this in the first place. I guess on a second glance I see things to change.

Reply:
😄 It happens to us all. I do like your suggested changes, first giving a shot at abstracting it.

Reply:
There were more bugs uncovered while trying to abstract. Opened a new issue, #887 to address it separately.